### DISCUSSION TITLE
Is it possible to use the dynamicWildcard function?

### DISCUSSION THREAD

**Discussion by 1025307336** [mcmonkeyprojects/SwarmUI#277] [General]
Or is there a way to use the text of the prompt output from the comfyui workflow in the generated page, just the text, and continue to debug the other features as you please?

**Comment by mcmonkey4eva** [comment]
Huh?

**Reply by yo1nkers** [reply]
I believe I understand what they were trying to say.

Automatic1111 has an extension called Dynamic Prompts and you can use "dynamic wildcards" within the extension, which are a much more detailed method of wildcard application which allows one single wildcard and maximum creativity, rather than selecting predefined wildcard topics.

Whilst you can always just do <wildcard:> in the prompt, I'm assuming they're asking for more.

**Reply by mcmonkey4eva** [reply]
That's... not really helpful.
`I'm assuming they're asking for more.` what they're asking for is the thing that needs to be clarified.
I also don't think any part of their message implied they want _more than the current wildcard system_. They might very well be asking how to use wildcards just at all. But the message asks about comfyui workflows and feature debugging, which is potentially something way more complex and precise.
So let's let them explain what they're trying to ask.
